docs/csv/example_files/candidate_contest.txt

```diff
@@ -1 +1,3 @@
-id,abbreviation,candidate_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,number_elected,office_ids,primary_party_ids,votes_allowed
\ No newline at end of file
+id,abbreviation,ballot_selection_ids,ballot_sub_title,ballot_title,electoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,number_elected,office_ids,primary_party_ids,votes_allowed
+cancon001,SE-1,"bs001 bs002”,,Governor of Virginia,ed001,all registered voters,fips,,49,true,Governor,1,,,1,"off001","par01”,1
```

Comment:
Some elements are quoted, others aren't. Is there a particular rhyme or reason to that?